internal/pkg/agent/application/pipeline/actions/handlers/handler_action_unenroll.go

```diff
@@ -75,7 +75,9 @@ func (h *Unenroll) Handle(ctx context.Context, a fleetapi.Action, acker store.Fl
 	} else if h.stateStore != nil {
 		// backup action for future start to avoid starting fleet gateway loop
 		h.stateStore.Add(a)
-		h.stateStore.Save()
```

Comment:
linter complained: Error return value of `h.stateStore.Save` is not checked (errcheck)

Reply:
do you prefer a `// nolint: errcheck` instruction?

Reply:
Lets do the nolint here, I am not sure of the impact of this changes here, wdyt @michel-laterman ?

Reply:
I think that currently `state.Action` can be either unenroll or policy-change. (this is different from the action queue part I added).

I think if the action fails to persist, then the agent will try to start with the persisted policy if it's restarted. But at that point it's api key has likely been revoked.

Reply:
updated the PR to use the `nolint` instruction.